go/enclave/components/batch_registry.go

```diff
@@ -148,6 +148,7 @@ func (br *batchRegistry) GetBatchAtHeight(height gethrpc.BlockNumber) (*core.Bat
 			return nil, fmt.Errorf("batch with requested height %d was not found. Cause: %w", height, err)
 		}
 		batch = headBatch
+	case gethrpc.SafeBlockNumber, gethrpc.FinalizedBlockNumber:
```

Comment:
Looks like this returns `nil, nil` at the mo, is that intentional? I guess it should return an 'unsupported' error if we're not going to handle it just yet.

Reply:
Not sure I follow, I just added these as default behavior to avoid the lint error.
Should I add a condition to return 'unsuported' ?

Reply:
They don't fall through to default though, golang's weird with that isn't it (https://go.dev/play/p/SjqwN7NZflI)

so anyone calling with that param will get nil, nil back. I'd suggest adding these two to the one above like:

```
// for now we are treating `finalized` and `safe` heads the same as latest head
case gethrpc.LatestBlockNumber, gethrpc.SafeBlockNumber, gethrpc.FinalizedBlockNumber:
```

Reply:
Yeah now I see what you mean, I forget to use the `fallthrough` keyword.

But that suggestion makes sense, fixed 👍